util/git/client.go

```diff
@@ -353,17 +376,15 @@ func (m *nativeGitClient) LsRemote(revision string) (string, error) {
 	// symbolic reference (like HEAD), in which case we will resolve it from the refToHash map
 	refToResolve := ""
 	for _, ref := range refs {
-		refName := ref.Name().String()
-		if refName != "HEAD" && !strings.HasPrefix(refName, "refs/heads/") && !strings.HasPrefix(refName, "refs/tags/") {
-			// ignore things like 'refs/pull/' 'refs/reviewable'
+		shortRefName, ok := getShortRefName(remote, ref.Name())
+		if !ok {
+			// log.Debugf("ignoring ref %s", ref.Name().String())
```

Comment:
This was basically inspired from the previous commented log statement `//log.Debugf("%s\t%s", hash, refName)`.

I can un-comment it, but I think this may produce loads of log lines, depending on your fetch refspec. But okay, it's a debug log ;) WDYT?

Reply:
meh - leave it